src/components/data-view/data-view.js

```diff
@@ -0,0 +1,122 @@
+import React, { Component } from 'react'
+import colors from '../../styles/variables'
+import './data-view.scss'
+import ReactTable from 'react-table'
+import 'react-table/react-table.css'
+import ReactJson from 'react-json-view'
+import LoadingElement from '../../components/generic-elements/loading-element'
+import { Tab, Tabs, TabList, TabPanel } from "react-tabs";
+import _ from 'lodash'
+
+export default class extends Component {
+  constructor() {
+    super();
+    this.state = { index: 0 };
+  }
+
+  render() {
+    const isGeojson = this.props.format == 'geojson'
```

Comment:
Is this a re-organization? Is there no way to make sure we clear the data in redux before and after a component is mounted? I know this is how we clean up query results, at least